.gitignore

```
@@ -3,6 +3,11 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+# Exclude source archives from the spec directory
+SPECS/**/*.tar.*
+SPECS/**/*.tgz
+SPECS/**/*.zip
```

Good idea! Could you also remove any existing archives, which accidentally got into our repo as well? I was able to find only two right now:
- `SPECS/telegraf/telegraf-vendor-1.14.5.tar.gz` and
- `SPECS/zipper/minizip.tar.gz`.

```suggestion
SPECS/**/*.xz
SPECS/**/*.zip
```

---

We could also [embrace what others have already came up with in this space](https://github.com/github/gitignore/blob/master/Global/Archives.gitignore).

---

Oh awesome, that list is way more thorough. I'll go ahead and implement that instead. Will remove existing archives as well (assuming they're not load-bearing mistakes, somehow).

---

Removed the existing archives, and checked that they're in the source blob.